src/apps/commands/management/commands/generate_data.py

```diff
@@ -10,15 +10,25 @@ class Command(BaseCommand):
     def add_arguments(self, parser):
         # Optional argument
         parser.add_argument(
-            '-s'
+            '-n',
+            '--no-admin',
+            type=bool,
+            help='Whether to create an admin'
```

Comment:
Why does this default to true? shouldn't it default to false, and then we add the `--no-admin` flag if we _don't_ want an admin account? If we want the default behavior to be not creating an admin account, we should rename this command to `--create-admin` or something so it is additive instead of exclusive, and so it is much more intuitive.